src/components/CallbackForm/CallbackForm.js

```diff
@@ -28,32 +28,15 @@ export default function CallbackForm({residentId, resident, backHref, saveFuncti
     const [errorsExist, setErrorsExist] = useState(null)
 
     const spokeToResidentCallOutcomes = [
-        {
-            label: "Callback complete",
-            value: "callback_complete"
-        },
-        {
-            label: "Refused to engage",
-            value: "refused_to_engage"
-        },
-        {
-            label: "Call rescheduled",
-            value: "call_rescheduled"
-        }
+        {name: "Callback comeplete" ,value: "callback_complete"},
+        {name: "Refused to engage" ,value: "refused_to_engage"},
+        {name: "Call rescheduled" ,value: "callback_complete"},
+
     ];
     const noAnswerCallOutcomes = [
-        {
-            label: "Voicemail left",
-            value: "voicemail"
-        },
-        {
-            label: "Wrong number",
-            value: "wrong_number"
-        },
-        {
-            label: "No answer machine",
-            value: "no_answer_machine"
-        }
+        {name: "Voicemail left" ,value: "voicemail_left"},
```

Comment:
I think we've been saving this as 'voicemail' in the db

Reply:
yeah your right, ive updated this now [here](https://github.com/LBHackney-IT/coronavirus-here-to-help-frontend/pull/31/commits/525c0d03a3a424e1f12cf2b47c02d85512df2d95)